### PR TITLE
DTSERWFOUR-463-Allow Transaction Details text to be selected

### PR DESCRIPTION
**Pull request by shanavas123**

### CODE REVIEW

File: receiptui/src/main/res/layout/fragment_receipt_detail.xml [hunk 1]
```diff
@@ -43,7 +46,62 @@
...
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="@string/text_placeholder"
+                        style="@style/ReceiptCurrency"
+                        android:textIsSelectable="true"
+                        android:focusable="true"
+                        android:longClickable="true"
+                        app:layout_constraintBottom_toBottomOf="parent"
+                        app:layout_constraintEnd_toEndOf="parent"
+                        app:layout_constraintTop_toBottomOf="@+id/transaction_title" />
```
Comment (rbanikPP) [inline receiptui/src/main/res/layout/fragment_receipt_detail.xml:105]: why this section is added?

Comment (shanavas123) [reply receiptui/src/main/res/layout/fragment_receipt_detail.xml:105]: <include layout="@layout/receipt" /> common for recent receipt, transaction list.

**Review by rbanikPP — APPROVED** [review]

File: receiptui/src/main/res/layout/fragment_receipt_detail.xml (hunk 1 above)
Comment (vwagh-hw) [reply receiptui/src/main/res/layout/fragment_receipt_detail.xml:102]: can you explain why `<include layout="@layout/receipt" />` had to be deleted and replaced with 100 lines

Comment (shanavas123) [reply receiptui/src/main/res/layout/fragment_receipt_detail.xml:102]: <include layout="@layout/receipt" /> is common for home recent recipe list and transaction recipe list ,while implement the text selection its reflect in home and transaction list.

Comment (shanavas123) [reply receiptui/src/main/res/layout/fragment_receipt_detail.xml]: revert the `<include layout="@layout/receipt" />` text selection added in code

**Review by vwagh-hw — APPROVED** [review]

**Pull request merged**